src/hooks/useHashLocation.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseLocationHook } from 'wouter'
+import { useEffect, useState } from 'preact/hooks'
+
+// returns the current hash location in a normalized form
+const currentLocation = () => {
```

Comment:
We use location in some places, make sure they work without this function

Reply:
`location` works well. Checked on `preview` & `start`